website/content/en/docs/concepts/_index.md

```diff
@@ -0,0 +1,64 @@
+
+---
+title: "Intro to Karpenter Concepts"
+linkTitle: "Concepts"
+weight: 20
+resources:
+- src: "**fig1*.png"
+- src: "**fig2*.png"
+---
+
+Kubernetes is designed to schedule new pods onto existing nodes. However, when existing nodes get full, Kubernetes struggles with automatically provisioning new nodes from a cloud provider.
+
+Karpenter activates when pending pods are unable to be fit onto existing nodes. Karpenter understands the instance types available from cloud providers, such as AWS. Karpenter’s provisioner intelligently selects new instances to start, using rich data from both podspecs and the cloud provider APIs (such as EC2’s flexible fleet API). 
+
+Karpenter balances responding rapidly to un-schedulable pods and making efficient provisioning decision. Karpenter is substantially faster than the cluster autoscaler, while additionally considering podspec labels and cloud provider availability zones. Specifically, the commonly used kubernetes properties such as labels, taints, affinity (node, pod), and anti-affinity are supported.
+
+Activating Karpenter on your cluster is a combination of a helm chart, and configuring the cloud platform to accept provisioning requests from Karpenter. On AWS, IAM roles for Service Accounts (IRSA) is used. The Kubernetes control plane in EKS oversees cluster-space Karpenter requests being securely elevated and passed on to the cloud platform. 
+
+## Provisioner CRD
+
+[Provisioner]({{< ref "/docs/provisioner-crd.md" >}}) is the primary Custom Resource Definition (CRD) for Karpenter, and you need at least one. 
+
+The primary kubernetes API object for Karpenter is “provisioner”. Notably, one provisioner can handle multiple node profiles (graphics enabled, compute optimized, memory optimized, etc). Karpenter is group-less, and eliminates management of multiple node groups with fixed instance specs. In short, the Karpenter provisioner object is focused on podspecs. This simplifies cluster management, and reduces the complexity of implementing Karpenter. 
+
+## Provsioning Procedure
+
+Karpenter activates when a pod is unschedulable. In response, it starts provisioning a new node, including the cloud provider instance. 
+
+Karpenter makes new instances based on the Provisioner CRD. In the default configuration, Karpenter automatically selects from all the general purpouse instances available, with an intent to accomodate the unscheduled pods efficently. Alternatively, the Provisioner CRD can define a list of acceptable instance types. 
+
+The Provisioner CRD includes lables for the new nodes. Certain labels, such as `kubernetes.io/arch`, change the instances Karpetner will provision. For example, setting `kubernetes.io/arch=arm64` configures Karpenter to provision ARM based instances. 
```

Comment:
Worth mentioning that every node created by a provisioner is labeled as such
`karpenter.sh/provisioner-name: foo`